kotlinx-coroutines-core/common/src/channels/Channel.kt

```diff
@@ -583,9 +1065,11 @@ public interface ChannelIterator<out E> {
      * See [suspendCancellableCoroutine] for low-level details.
      *
      * Because of the prompt cancellation guarantee, some values retrieved from the channel can become lost.
-     * See "Undelivered elements" section in [Channel] documentation for details on handling undelivered elements.
+     * See the "Undelivered elements" section in the [Channel] documentation
+     * for details on handling undelivered elements.
      *
-     * Note that this function does not check for cancellation when it is not suspended.
+     * Note that this function does not check for cancellation when it is not suspended, that is,
+     * if the next element is immediately available.
      * Use [yield] or [CoroutineScope.isActive] to periodically check for cancellation in tight loops if needed.
```

Comment:
```suggestion
     * Use [ensureActive] or [CoroutineScope.isActive] to periodically check for cancellation in tight loops if needed.
```
Other places were changed to refer to `ensureActive` instead of `yield`.